### ISSUE TITLE
Can no longer add server with Tailscale

### ISSUE THREAD

**Issue by exstntlmsnthrp** [lollipopkit/flutter_server_box#610]
**Describe the bug**

Up through app version 1.0.1051 you can add servers by having Tailscale installed on the phone and server. It does not require a password or authentication as that is handled by Tailscale. On all updates since then, this feature is broken and provides a timeout error. 

**To Reproduce**

**Desired Results**
 Desired result is to continue using server box with Tailscale as authentication. 

**Actual Results**

I can no longer connect to servers. 

**Screenshots**

Version 1051:
![Screenshot_20241021-212407.png](https://github.com/user-attachments/assets/fe655ae7-0d76-47cc-b114-d8107c76d960)

Version 1104:
![Screenshot_20241021-212556.png](https://github.com/user-attachments/assets/0bf2a689-3091-40bf-a4d1-bb2a13b39d91)

**Device**
 Android 14 and Android 15 beta
App version 1051 and 1104

**Additional context**

**Comment by AstroEngineeer** [comment]
I'm facing this issue as well. Since i just installed ServerBox yesterday I thought it could be a problem from my side

**Comment by exstntlmsnthrp** [comment]
> I'm facing this issue as well. Since I just installed ServerBox yesterday I thought it could be a issue from my side but couldn't find any.
> 
> The error I'm getting:
> 
> Authentication failed, please check whether credentials are correct
> 
> ```sh
> SSHAuthAbortError(Connection closed before authentication: null)
> 

Yes, this is the problem I'm having as well. It never occurred with version 1051 or earlier. Only from versions after this. 

You can download version 1051 from GitHub or from F-Droid and it works fine. But remember to disable update notifications.

**Comment by BSoDium** [comment]
Same here, the provided user and port are correct, but this error pops up and the app crashes when attempting SSH connection

**Comment by exstntlmsnthrp** [comment]
Will this be fixed in the new release?

**Comment by exstntlmsnthrp** [comment]
Still having this same error. Will this get fixed? The old version that was previously working no longer works on my version of Android (16 beta) and I was required to update. Now I cannot connect because of the ```
SSHAuthAbortError(Connection closed before authentication: null)``` error.

**Comment by GT-610** [comment]
Latest version (v1297) works well with Tailscale servers. Please check it out.
